aws/docker/worker/generate_profile_plot.py

```diff
@@ -156,11 +156,18 @@ def get_plot(x, y, z, cmap='cmap', title='Glider Profiles', ylabel='Pressure (db
     if len(z[np.logical_not(np.isnan(z))]) == 0:
         ax.set_ylim(0, 10)
         # annotate the plot to report data issue
+<<<<<<< HEAD
```

Comment:
Can you remove the remnants of the merge conflict please?

Reply:
Fixed and committed changes.